modules/virt-querying-the-node-exporter-service-for-metrics.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-querying-the-node-exporter-service-for-metrics-_{context}"]
+= Querying the node-exporter service for metrics
+
+Metrics are exposed for virtual machines through an HTTP service endpoint under the `/metrics` canonical name. When you query for metrics, Prometheus directly scrapes the metrics from the metrics endpoint exposed by the virtual machines and presents these metrics for viewing.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges or the `monitoring-edit` role.
+* You have enabled monitoring for the user-defined project by configuring the `node-exporter` service.
+* You configured the `ServiceMonitor` resource that scrapes the metrics exposed by the `node-exporter` service.
+
+.Procedure
+* To list all available metrics for the `node-exporter` service, use the `curl` query against `http://<endpoint>/metrics`.
++
+[source,terminal]
+----
+$ curl http://<172.30.226.162.9100>/metrics | grep -vE "^#|^$"
```

Comment:
Update with this --> `$ curl http://<172.30.226.162:9100>/metrics | grep -vE "^#|^$"` (Colon was missing before `9100`)

Reply:
Updated the content. Please review.